bag/datasets/brk/tests/test_sensitive_details_jwt.py

```diff
@@ -33,12 +36,15 @@ def setUp(self):
 
         now = int(time.time())
 
-        token_default = jwt.encode(
-            {'authz': authorization_levels.LEVEL_DEFAULT, 'iat': now, 'exp': now+600}, key, algorithm=algorithm)
-        token_employee = jwt.encode(
-            {'authz': authorization_levels.LEVEL_EMPLOYEE, 'iat': now, 'exp': now+600}, key, algorithm=algorithm)
-        token_employee_plus = jwt.encode(
-            {'authz': authorization_levels.LEVEL_EMPLOYEE_PLUS, 'iat': now, 'exp': now+600}, key, algorithm=algorithm)
+        token_default = jwt.encode({
+            'authz': authorization_levels.LEVEL_DEFAULT,
+            'iat': now, 'exp': now+600}, key, algorithm=algorithm)
+        token_employee = jwt.encode({
+            'authz': authorization_levels.LEVEL_EMPLOYEE,
+            'iat': now, 'exp': now+600}, key, algorithm=algorithm)
+        token_employee_plus = jwt.encode({
+            'authz': authorization_levels.LEVEL_EMPLOYEE_PLUS,
+            'iat': now, 'exp': now+600}, key, algorithm=algorithm)
```

Comment:
Pep8 klaagt over het gebrek aan whitespace rondom het plusje: `now+600` vs `now + 600` (dit zat er blijkbaar al in voor jouw change)